### PR TITLE
audit minor updates

### PR DESCRIPTION
**Pull request by xiaozhou**

### CODE REVIEW

File: scripts/solc_abigen.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ solFiles=(
 )
 
 dld_solc() {
-  curl -L "https://binaries.soliditylang.org/linux-amd64/solc-linux-amd64-${SOLC_VER}" -o solc && chmod +x solc
+  curl -L "https://github.com/ethereum/solidity/releases/download/${SOLC_VER}/solc-static-linux" -o solc && chmod +x solc
```
Comment (Dominator008) [inline scripts/solc_abigen.sh:30]: Why this change? I thought binaries.soliditylang.org is the recommended URL.

Comment (xiaozhou) [reply scripts/solc_abigen.sh]: updated

**Review by Dominator008 — APPROVED** [review]
